src/CsharpClient/QuixStreams.Streaming/QuixApi/Portal/Workspace.cs

```diff
@@ -176,4 +181,194 @@ internal enum WorkspaceStatus
         /// </summary>
         Disabled
     }
+    
+    internal enum WorkspaceBrokerType
+    {
+        /// <summary>
+        /// Quix shared managed Kafka cluster
+        /// </summary>
+        SharedKafka,
+
+        /// <summary>
+        /// Confluent Cloud Kafka cluster 
+        /// </summary>
+        ConfluentCloud,
+
+        /// <summary>
+        /// Self Hosted Kafka cluster
+        /// </summary>
+        SelfHosted,
+
+        /// <summary>
+        /// Amazon Msk Kafka cluster
+        /// </summary>
+        AmazonMsk
+    }
+    
+    /// <summary>
+    /// Describes the workspace broker settings
+    /// </summary>
+    internal class WorkspaceBrokerSettings
+    {
+        /// <summary>
+        /// Broker type
+        /// </summary>
+        public WorkspaceBrokerType BrokerType { get; set; }
+
+        /// <summary>
+        /// Syncronize existing topics in the broker
+        /// </summary>
+        public bool SyncTopics { get; set; }
```

Comment:
Lets limit the config to only what we need for now. Which is BrokerType and ClientId from the confluent type.